h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoostModel.java

```diff
@@ -265,15 +264,8 @@ public void dump(String format) {
   public static BoosterParms createParams(XGBoostParameters p, int nClasses, String[] coefNames) {
     Map<String, Object> params = new HashMap<>();
 
+    params.put("nround", p._ntrees);
     // Common parameters with H2O GBM
-    if (p._n_estimators != 0) {
```

Comment:
if `_n_estimators` worked before we shouldn't remove it in a fix release, this is okay to do in master (and make a note in changelog - or migration document)